### PR TITLE
#18 Add ability to change hand color.

### PR DESCRIPTION
**Pull request by Mehequanna**
This adds the ability to update the hand to be either black or red. Test by going into the camera view and press the black and red button on the bottom left.

### CODE REVIEW

**Review by kjanderson2 — CHANGES_REQUESTED** [review]
You're gonna want to hide these buttons when after the capture occurs or you end up having them weirdly below the back button.

**Comment by Mehequanna** [comment]
> You're gonna want to hide these buttons when after the capture occurs or you end up having them weirdly below the back button.

@kjanderson2 Oh my gosh, thank you for catching that! How did I forget that?

File: Android/app/src/main/res/layout/activity_mmiw_ar.xml [hunk 1]
```diff
@@ -86,6 +88,28 @@
...
+        android:layout_width="36dp"
+        android:layout_height="36dp"
+        android:layout_marginStart="32dp"
+        android:background="@drawable/color_button_black"
+        android:contentDescription="@string/change_to_black_hand_color"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@+id/capture_button" />
+
+    <Button
```
[the remark this answers is not in the document]
Comment (Mehequanna) [reply Android/app/src/main/res/layout/activity_mmiw_ar.xml:102]: @kjanderson2 do you know what I can do to complete this task. I'm all for accessibility, but I don't know how to name or label a button.

[the remark this answers is not in the document]
Comment (kjanderson2) [reply Android/app/src/main/res/layout/activity_mmiw_ar.xml:102]: This is odd... a contentDescription is an accessible label so I'm not sure what more it wants from you there... Is accesslint made for Android? Seems odd that it's not recognizing that as the right way to do accessibility for a button

**Review by kjanderson2 — APPROVED** [review]
Tested it out and it works great! I still like the red hand best though :)

**Pull request merged**